js/exercise-04.js

```diff
@@ -0,0 +1,33 @@
+/*
+    What would each of these lines of code do:
+
+    1. true && console.log("moo");
+    2. false && console.log("moo moo?")
+    3. true || console.log("hello friend");
+    4. false || console.log("bye friend");
+
+    Once you see the results, write in a comment why it works that way
+*/
+true && console.log("moo"); 
+/* output: moo
+Since we have a true boolean value and it executes console.log(), therefore "moo" is printed to the console
+*/
+false && console.log("moo moo?"); 
+/* output: false
+Since the left operand is a false boolean value and the console.log() it is executed,
+so false AND true = false, it will print to the console false
```

Comment:
This assumption is not completely right.

Are u sure this will log `false`?

FIX: execute script and check behaviour. Update explanation.